### ISSUE TITLE
Is refgenie still being actively developed?

### ISSUE THREAD

**Issue by birnbera** [refgenie/refgenie#274]
Refgenie looks like a great project that solves a very important problem in production bioinformatics work. Coworkers have given it great reviews. I would love to adopt it to manage a bunch of genomes (duh) however I'm seeing that development has virtually ceased since the end of last year. In the docs there are several allusions to future work allowing users to provide custom scripts to build various types of references that might not be included out of the box. This is something I'd be particularly interested in, however I don't want to adopt the project and hold out hope for that feature if refgenie is no longer being actively developed/maintained. Any insight into the future plans would be greatly appreciated. Thanks!

**Comment by nsheff** [comment]
Hey @birnbera thanks for the question. 

Development will continue and the project will likely increase in scope in the coming months/years.

We did slow down a bit over the past few months due to a developer leaving, and waiting on grant feedback. But anyway, the project is going forward now and I expect an increase in activity over the next 6-12 months as we push for a new release and another grant submission this fall. We're also starting to finalize integration with nextflow so development is starting to increase again, and has over the past few weeks. The custom scripts features you mention are implemented in development branches, and this is what we'll be working on further in the next year or so. So, you can probably already start to make use of it if you're willing to adopt the developmental branch and help test it out.

**Comment by lparsons** [comment]
It doesn't look like the planned increase in activity actually happened. I too like the concept of the project, but I don't want to invest too heavily if it is no longer being maintained. Any updates?

**Comment by nsheff** [comment]
Hi @lparsons, thanks for the interest. Basically, development is slow as I wait on funding decisions to try to continue the project. I'm still planning to continue, and I have lots of ideas for improvements I'm hoping to attract some external support to help get refgenie to the next level. Part of my attention has shifted to [GA4GH sequence collections](https://seqcol.readthedocs.io/en/latest/), which is a related project that spun off of refgenie for me, and on which refgenie will depend on in the future. I have been needing to get that work in a sense "finalized", since that is a foundational way to refer to reference genomes that would introduce some breaking changes into refgenie, so this has caused me to pause some internal focus on refgenie in the meantime. The good news is that project is nearing a point of publication that would enable refgenie to move forward in the coming months.

So, to make a long story short: I definitely haven't abandoned refgenie. I'm planning a major update that will greatly expand the refgenie capabilities, and I've been working on related projects to set the stage for that. I am not sure of the timing of this, as it will be sped up if I can get funding for the project. In the meantime, it will probably continue slowly. I'd be delighted to have any help and would be happy to talk if you or anyone is interested in contributing.

**Comment by lparsons** [comment]
Thanks so much for the thorough and quick reply. I certainly understand having limited time and multiple priorities. Thanks so much for everything you have done so far, it's very much appreciated. Best of luck going forward. I'll continue to keep an eye on things here and contribute if/when I can. Looking forward to many great things to come.

**Comment by nsheff** [comment]
Closing this issue. We have been working hard behind the scenes and have several updates that will be released in the coming days. 

The GA4GH Sequence Collections standard was approved by GA4GH in 2025. Since then, we've developed a new refgenie version from the ground up, that is based on this new standard.

The current refgenie/refgenconf/refgenieserver packages will get one final modernization release, and then will go into maintenance mode. A new refgenie version will be released shortly thereafter with many upgrades over the original.

So, keep an eye out for new releases in the coming weeks.
